tests/commands/record/test_session.py

```diff
@@ -50,3 +65,53 @@ def test_run_session_with_observation(self):
         payload = json.loads(responses.calls[0].request.body.decode())
         self.assert_json_orderless_equal(
             {"flavors": {}, "observation": True}, payload)
+
+    @responses.activate
+    @mock.patch.dict(os.environ, {
+        "LAUNCHABLE_TOKEN": CliTestCase.launchable_token,
+        JENKINS_URL_KEY: "https://jenkins.example.com/",
+        JENKINS_BUILD_URL_KEY: "https://jenkins.example.com/job/launchableinc/job/example/357/",
+        'LANG': 'C.UTF-8',
+    }, clear=True)
+    def test_run_session_with_jenkins_url(self):
+        result = self.cli("record", "session", "--build",
+                          self.build_name, "--observation")
+        self.assertEqual(result.exit_code, 0)
+
+        payload = json.loads(responses.calls[0].request.body.decode())
+        self.assert_json_orderless_equal(
+            {"flavors": {}, "observation": True, "link": {"provider": "jenkins", "url": "https://jenkins.example.com/job/launchableinc/job/example/357/"}}, payload)
+
+    @responses.activate
+    @mock.patch.dict(os.environ, {
+        "LAUNCHABLE_TOKEN": CliTestCase.launchable_token,
+        GITHUB_ACTION_KEY: "1",
```

Comment:
nits:

`GITHUB_ACTIONS` and `CIRCLECI` will be set `true` as a bool value. How about aligning the value type?